code/drasil-build/package.yaml

```diff
@@ -12,6 +12,9 @@ extra-source-files: []
 dependencies:
 - base >= 4.7 && < 5
 - pretty
+- text
+- word-wrap
+- drasil-codeLang
```

Comment:
I think you will need to add `drasil-codeLang` to `drasil-build`s `stack.yaml` now too.

Reply:
Ok I will do that right now!